src/values.rs

```diff
@@ -71,10 +71,8 @@ impl<T, S> Values<T, S> {
         }
     }
     
-    /// An iterator visiting all elements in arbitrary order.
-    ///
-    /// The iterator element type is &'a T.
-    pub fn single(&self) -> Option<&T> {
+    /// Returns the first element.
```

Comment:
I think we may want to expand on this a little. For example, what does "first" mean if the backing store is a `HashBag`?